packages/design/src/components/calendar/components/dialogs/confirm-appointment-move.tsx

```
@@ -0,0 +1,158 @@
+        pendingNewStartMinutes,
+        setPendingNewStartMinutes,
+        setNewStartTime,
+        originalAppointment,
+        setOriginalAppointment
+    } = useCalendarStore();
+
+    const dentists = useCalendarStore(state => state.dentists);
+    const newStartTime = useCalendarStore(state => state.newStartTime);
+    const setAppointments = useCalendarStore(state => state.setAppointments);
+    const getAppointmentDuration = useCalendarStore(state => state.getAppointmentDuration);
+    const getAppointmentDuration = useCalendarStore(state => state.getAppointmentDuration);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getAppointmentDuration'. Consider to delete it or rename it.

'getAppointmentDuration' is defined here:


(lint/suspicious/noRedeclare)